CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+- [FIXED] Raise exception in `create_database` when database name is invalid.
```

Comment:
Based on my recommended change I think this would need to be updated to something more general like:
`- [FIXED] Correctly raise exceptions from `create_database` calls.`